File: working/iosApps/SharedSwift/ModelKotlinExtensions/Player+UI,DTO+UI.swift
//
//  Player+UI.swift
//  MiniMate
//
//  Created by Garrett Butchko on 3/8/26.
//
import SwiftUI
import shared

extension Player {
    var ballColor: Color {
        ballColorDT?.toColor() ?? .mainOpp
    }
}

extension PlayerDTO {
    var ballColor: Color {
        ballColorDT?.toColor() ?? .mainOpp
    }
}

File: working/shared/src/androidMain/kotlin/com/garrettbutchko/minimate/ModelKotlinExtensions/Player+UI,DTO+UI.kt
package com.garrettbutchko.minimate.ModelKotlinExtensions

import androidx.compose.ui.graphics.Color
import com.garrettbutchko.minimate.datamodels.Player
import com.garrettbutchko.minimate.datamodels.PlayerDTO

val Player.ballColor: Color
    get() = ballColorDT.toColor()


val PlayerDTO.ballColor: Color
    get() = ballColorDT.toColor()

